GliaWidgetsTests/Sources/Coordinators/EngagementCoordinator/EngagementCoordinatorCallTests.swift

```diff
@@ -0,0 +1,136 @@
+import Foundation
+import XCTest
+@testable import GliaWidgets
+
+extension EngagementCoordinatorTests {
+    // Start
```

Comment:
if we would like to separate tests by sections, would it make sense to use pragma marks?
```
// MARK: - Start
```

Reply:
Fixed 6964ad8bae2e13f49b75f2e6cfee601c7ca2c4a1